stimela/cargo/cab/autoflagger/src/run.py

```diff
@@ -20,7 +20,7 @@
 strategy = jdict.pop("strategy", None)
 if strategy:
     strategy = utils.substitute_globals(strategy) or INPUT + "/" + strategy
-strategy = "-strategy %s/%s"%(INPUT, strategy) if strategy else ""
+strategy = "-strategy %s"%(strategy) if strategy else ""
```

Comment:
@bennahugo Is the strategy file baked into the container here? Should it not be in the INPUT directory along with other files that are configurable?